modules/persistence/src/services/metadata/index.ts

```diff
@@ -23,3 +23,13 @@ export const insertMetadata = async (buildId: ObjectId, zip: AdmZip) => {
     throw error;
   }
 };
+
+export const insertUmbrellaMetadata = async (buildId: ObjectId, zip: AdmZip) => {
+  try {
+    const umbrellaMetadata = await mergeAssociatedToCs(metadataFromZip(zip)[0]);
+    return umbrellaMetadata ? insert([umbrellaMetadata], COLLECTION_NAME, buildId) : undefined;
```

Comment:
getting an error here for same document._id. should strip, or upsert?

Reply:
This is now stripping the `_id` field, since we want to treat it as a new document entirely 👍 - thanks for flagging!